src/App.tsx

```diff
@@ -1,10 +1,51 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useCallback, useEffect, useMemo, useState,
+} from 'react';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { Footer } from './components/Footer';
+import { FilterBy } from './types/Filter';
+import { TodoList } from './components/TodoList';
+import { filterTodo } from './helpers/filterTodo';
 import { UserWarning } from './UserWarning';
+import { Header } from './components/Header';
+import { ErrorType } from './types/Errors';
+import { Notifications } from './components/Errors';
 
-const USER_ID = 0;
+const USER_ID = 12065;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, steErrorMessage] = useState<ErrorType | null>(null);
+  const [filterBy, setFilterBy] = useState(FilterBy.All);
+
+  const visibleError = (error: ErrorType) => {
```

Comment:
```suggestion
  const handleError = (error: ErrorType) => {
```
A function name should start from the verb